### PR TITLE
UAR:1032 adding redirect urls tostartPaymentsSession

### PR DESCRIPTION
**Pull request by azafarCH**
### JIRA link
https://companieshouse.atlassian.net/jira/software/c/projects/UAR/boards/481?modal=detail&selectedIssue=UAR-1032

### Change description
Added redirect urls for startPaymentsSession to ensure redirect to correct submission confirmation pages.

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/controllers/shared/common.resume.submission.controller.ts [hunk 1]
```diff
@@ -51,7 +52,15 @@ export const getResumePage = async (req: Request, res: Response, next: NextFunct
             [config.PAYMENT_REQUIRED_HEADER]: config.PAYMENTS_API_URL + config.PAYMENTS
           }
         };
-        const redirectPath = await startPaymentsSession(req, session, transactionId, overseaEntityId, headersPaymentUrl);
+
+        let baseURL: any;
```
Comment (AndreKramer7654) [inline src/controllers/shared/common.resume.submission.controller.ts:56]: : string;

File: src/controllers/shared/common.resume.submission.controller.ts [hunk 2]
```diff
@@ -51,7 +52,15 @@ export const getResumePage = async (req: Request, res: Response, next: NextFunct
...
           }
         };
-        const redirectPath = await startPaymentsSession(req, session, transactionId, overseaEntityId, headersPaymentUrl);
+
+        let baseURL: string;
+        if (isRegistration){
+          baseURL = `${config.CHS_URL}${config.REGISTER_AN_OVERSEAS_ENTITY_URL}`;
+        } else {
+          baseURL = `${config.CHS_URL}${config.UPDATE_AN_OVERSEAS_ENTITY_URL}`;
+        }
```
Comment (RobertBromfield) [inline src/controllers/shared/common.resume.submission.controller.ts:61]: You can cut down the number of lines here by using a ternary operator 

```suggestion
     let baseURL = `${config.CHS_URL}${isRegistration ? config.REGISTER_AN_OVERSEAS_ENTITY_URL : config.UPDATE_AN_OVERSEAS_ENTITY_URL}`;

```

**Review by AndreKramer7654 — APPROVED** [review]

**Comment by AndreKramer7654** [comment]
Tested locally for no-change/change and registration. Looks good.

File: src/controllers/shared/common.resume.submission.controller.ts (hunk 2 above)
Comment (azafarCH) [reply src/controllers/shared/common.resume.submission.controller.ts]: wow, looks great, made the change!

**Review by bantao-ch — APPROVED** [review]

**Review by jkayyala — APPROVED** [review]

**Pull request merged**